scripts/model_fitting.py

```diff
@@ -0,0 +1,73 @@
+# model_fitting.py
```

Comment:
Follow-on comment from before, maybe should name file as `preprocessing_model_fitting.py` as our scaling is combined with pipeline.

Or alternatively, if we decided to strip out the scaler step separately to follow Tiff's example, name can remain unchanged.

In addition, the run commands for this script did not run successfully for me so have not inspected the scripts and resulting outputs (i.e. models and tables).